artiq/test/coredevice/test_embedding.py

```diff
@@ -117,6 +118,92 @@ def test_default_arg(self):
         self.assertEqual(exp.run(), 42)
 
 
+class _RPCOverflowTest(EnvExperiment):
+    def build(self):
+        self.setattr_device("core")
+
+    @kernel
+    def run(self, f, g):
+        f(g())
+
+
+class RPCOverflowTest(ExperimentCase):
```

Comment:
There could be many "overflows" with RPC. Rename to RPCIntOverflowTest?

Reply:
I've change the name to RPCHostRoundtripTypeTest, it should sound a lot better